src/library/constants.cpp

```diff
@@ -158,6 +158,7 @@ name const * g_int_neg_ne_of_pos = nullptr;
 name const * g_int_ne_neg_of_pos = nullptr;
 name const * g_int_neg_ne_zero_of_ne = nullptr;
 name const * g_int_zero_ne_neg_of_ne = nullptr;
+name const * g_interactive_executor = nullptr;
```

Comment:
@cipher1024 @khoek The `constants.*` files should not be edited by hand, they are automatically generated from the `constants.txt` file using the `scripts/gen_constants_cpp.py` script.

Reply:
Oh, ok, thanks for the heads up. When I wrote this I was totally on my own in the codebase. :)

Reply:
No hard feelings.  I was just surprised when I added something to `constants.txt` so I felt like I should share this.